docs/Displaying a User Interface/Main Screen Templates/index.md

```diff
@@ -134,6 +134,255 @@ if (success) {
 ```
 !@
 
+@![iOS]
+When changing screen layouts or template fields, it is highly recommended to encapsulate these updates into a class or method. Doing so is a good way to keep SDL UI changes organized. An example of this can be seen in the [example weather app](https://github.com/SmartDeviceLink-Examples/example_weather_app_ios). See below for another example:
+
+|~
```

Comment:
This code block is too long and has no explanatory comments. I think we need to split this up, but doing so for both objc and swift will be a bit tricky since swift is one file and objc is two.

1. I think we can definitely separate the protocol. We should have a small (markdown) title with a paragraph before it explaining what it is and does.
2. The home screen / button can be another one, again with a markdown title. The objc `.h` and `.m` would be in one tab and the swift in another.
3. Please add stubs with `// TODO` for JavaScript and Java Suite.